cluster/k8s-multus-1.12.2/.kubeconfig

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
please add this to the git ignore file

Reply:
And please do the same for following .kubectl

Reply:
Done